WordPress/Classes/ViewRelated/Voice/VoiceToContentView.swift

```diff
@@ -171,6 +153,48 @@ private struct VoiceToContentRecordingView: View {
     }
 }
 
+private struct buttonRecord: View {
```

Comment:
(nit) Name should start with a capital letter.

Reply:
Oops good catch. Fixed! [633ef61](https://github.com/wordpress-mobile/WordPress-iOS/pull/23288/commits/633ef6155a3ce81ee14a46c60ac6fa8317ffb7a9)